src/capi/azext_capi/helpers/os.py

```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+
+def set_enviroment_variables(dic):
+    """
+    Sets key, value pair items of dictionary into enviroment variables
```

Comment:
```suggestion
    Sets the key and value items of a dictionary into environment variables.
```

Reply:
Actually I just noticed that `set_environment_variables` itself is misspelled (missing an "n") as well.